UserAgent.xcodeproj/project.pbxproj

```diff
@@ -5489,7 +5489,6 @@
 			baseConfigurationReference = 8EF4515D84F7B934A5318AEC /* Pods-Cliqz.adhoc.xcconfig */;
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon_Adhoc;
-				CLIQZ_URL_SCHEME = cliqz;
```

Comment:
It should probably be removed from Ghostery config as well.

Reply:
Fixed.